game.py

```diff
@@ -20,8 +20,34 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
-
+    # random_word_generator = RandomWord()
+    # snowman_word = random_word_generator.word(
+    # word_min_length=SNOWMAN_MIN_WORD_LENGTH, 
+    # word_max_length=SNOWMAN_MAX_WORD_LENGTH)
+
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        
+        if user_input in correct_letter_guess_statuses:
```

Comment:
Love this! In the near future you will see how searching for a `key` in a dictionary has a considerable advantage over searching for an element in a list (Time Complexity if you you are interested!)